javascript/webhooks-nodejs/README.md

```diff
@@ -26,33 +26,55 @@ https://ngrok.com/docs/getting-started/
+ngrok http 8080
+```
+
 ## Usage
 
-1. Create a file named **.env** in the root directory of this app. In the file, include the following line:
+After ngrok is successfully running, the next steps are to run the server (**index.js**), then establish a webhook using the provided webhook setup script (**createWebhook.js**).
+
+1. Create a file named **.env** in the root directory of this app. In the file, include the following lines:
 
 ```
 X_HOOK_SECRET=
+PAT=
 ```
 
-The Express server writes the `X-Hook-Secret` obtained from the [webhook handshake](https://developers.asana.com/docs/webhooks-guide#the-webhook-handshake) directly into this local file, allowing the `X-Hook-Secret` to persist across server restarts. As such, this server is designed to handle a single `X_HOOK_SECRET`, which is associated with a single webhook.
+You do not need to manually supply a value for `X_HOOK_SECRET`. The Express server writes the `X-Hook-Secret` obtained from the [webhook handshake](https://developers.asana.com/docs/webhooks-guide#the-webhook-handshake) directly into this local file, allowing the `X-Hook-Secret` to persist across server restarts. As such, this server is designed to handle a single `X_HOOK_SECRET`, which is associated with a single webhook.
 
 > **Note:** In a production environment, this value should be securely stored in a database. For security reasons, ensure that you never commit or expose this value in a public repository.
 
-For your convenience, a sample **.env.template** file is included in the root directory of this application.
+Enter your [personal access token](https://developers.asana.com/docs/personal-access-token) for `PAT`. For your convenience, a sample **.env.template** file is included in the root directory of this application.
 
 2. Start the webhook server:
 
 ```
 npm run dev
 ```
 
-3. Create a webhook by calling `POST /webhooks` and specifying given resource (e.g., a task). Feel free to use the [Postman Collection](https://developers.asana.com/docs/using-postman) or the [POST /webhooks](https://developers.asana.com/reference/createwebhook) page in the API Explorer to make your request.
+3. Edit the **createWebhook.js** setup script to include details for your [POST /webhooks](https://developers.asana.com/reference/createwebhook) request:
 
-> **Note:** In your request, make sure to set the [`target`](https://developers.asana.com/docs/webhook) parameter to your public ngrok domain instead of the example `localhost` domain, `http://localhost:8000/receiveWebhook`. This means you must replace `localhost:8000` with your unique ngrok domain. The `target` should be your ngrok server's "Forwarding" domain followed by `/receiveWebhook`. 
+```js
+// TODO: Replace these values with your target URI, object ID, filter, and resource type
+const targetUri = "https://your.url.here/receiveWebhook"; // The webhook server's public endpoint for receiving webhooks
+const objectId = "TASK_ID_HERE"; // The Asana object ID you want to track
```

Comment:
You'll probably want to change this since you changed this in the `createWebhook.js` also to make it consistent let's add the`<` and `>` around the placeholder value `"TASK_ID_HERE"` -> `<OBJECT_ID_HERE>`

Reply:
Fixed! I originally left it out so that developers are not mislead that `<` and `>` belong in the request payload.